src/MSAL.Common.props

```diff
@@ -204,7 +204,7 @@
     <None Update="$(PathToMsalSources)\Platforms\net45\SilentWindowsFormsAuthenticationDialog.cs" SubType="Form" />
   </ItemGroup>
 
-  <ItemGroup>    
+  <ItemGroup Condition="'$(OS)' != 'Unix'">    
```

Comment:
I have made similar changes as part of the "Interactive auth on xamarin.mac". There is a slightly better way to do this, i.e. Condition="$([MSBuild]::IsOsPlatform('Windows')), because the variable $(OS) will be set to Unix for both Mac and Linux (which is technically accurate, but we want to differentiate between Mac and Linux).

Anyway, when you rebase, just take my changes on this file.

Reply:
ok, I will take a look.